Cargo.toml

```diff
@@ -116,7 +116,7 @@ zeroize = { version = "1.5.7", default-features = false, features = ["zeroize_de
 
 # Oxide forks and repos
 dice-mfg-msgs = { git = "https://github.com/oxidecomputer/dice-util", default-features = false }
-gateway-messages = {git = "https://github.com/oxidecomputer/management-gateway-service", default-features = false, features = ["smoltcp"] }
+gateway-messages = {git = "https://github.com/oxidecomputer/management-gateway-service", branch = "update-rot-state-msg", default-features = false, features = ["smoltcp"] }
```

Comment:
Nit - let's merge the MGS branch and put this back on `main` before merging this PR.